src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -71,6 +77,27 @@
     private static final List<Author> AUTHOR_CONFIG_SPECIAL_CHARACTER_AUTHORS = Arrays.asList(
             FIRST_SPECIAL_CHARACTER_AUTHOR, SECOND_SPECIAL_CHARACTER_AUTHOR, THIRD_SPECIAL_CHARACTER_AUTHOR);
 
+    private static final String FIRST_SPECIAL_CHARACTER_DISPLAY_NAME = "Tay Fan Gao, Douya";
```

Comment:
The result will still contain the double quote?

Reply:
Should not contain double quotes. Resulting display name should be `Tay Fan Gao, Douya`

Reply:
Sorry, actually meant to tag the following line. Let's continue the discussion over there